cpp/benchmarks/tpch/utils.hpp

```diff
@@ -28,17 +30,104 @@
 #include <cudf/transform.hpp>
 #include <cudf/unary.hpp>
 
-#include <rmm/cuda_device.hpp>
+#include <rmm/device_uvector.hpp>
+#include <rmm/mr/device/cuda_async_memory_resource.hpp>
 #include <rmm/mr/device/cuda_memory_resource.hpp>
 #include <rmm/mr/device/device_memory_resource.hpp>
 #include <rmm/mr/device/managed_memory_resource.hpp>
 #include <rmm/mr/device/owning_wrapper.hpp>
 #include <rmm/mr/device/pool_memory_resource.hpp>
+#include <rmm/mr/device/prefetch_resource_adaptor.hpp>
+#include <rmm/mr/device/statistics_resource_adaptor.hpp>
 
+#include <cstdlib>
 #include <ctime>
 
+namespace {
```

Comment:
We shouldn't use anonymous namespaces in headers. See [C++ Core Guidelines SF.21: Don’t use an unnamed (anonymous) namespace in a header](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rs-unnamed).